sdf/1.10/joint.sdf

```diff
@@ -112,6 +112,24 @@
       </element>
 
     </element> <!-- End Limit -->
+
+    <element name="mimic" required="0">
+      <description>Specifies a linear equality constraint between the output of two joint
+        axes as mimic_angle = multiplier * (other_joint_angle - reference) + offset.</description>
+      <attribute name="joint" type="string" default="" required="1">
+        <description>Name of the joint to be mimicked</description>
+      </attribute>
```

Comment:
I think I'd like to add the `axis` attribute as well even if dart doesn't fully support it yet. We can print an error message in gz-physics about what is not yet supported

Reply:
added in https://github.com/gazebosim/sdformat/pull/1166/commits/d09a4320f3921066c0195921f7d320f9204732a8